### ISSUE TITLE
[Taskbar Thumbnail Reorder] explorer hung when drag grouped thumbnail

### ISSUE THREAD

**Issue by yoyo-real** [ramensoftware/windhawk-mods#29]
When try drag grouped thumbnail, Explorer was hung.
Will can fix it?

**Comment by m417z** [comment]
What do you mean?
You wrote here that the recent update fixed your problem: https://github.com/ramensoftware/windhawk-mods/issues/25#issuecomment-1193833875
Is there another problem you're seeing?

**Comment by yoyo-real** [comment]
Sorry for not detailed report.

> Is there another problem you're seeing?

Yes. Stil hangs after update, but hangs timing is different.

1. Open some windows same app.
2. Use tiling layout. Some windows are grouped.
3. Drag window thumbnail across grouped thumgnail, or drag grouped thumbnail.
4. Exporer has hung.

![image](https://user-images.githubusercontent.com/1130045/182051564-6f093843-f986-48ff-9a3c-c76070766757.png)

I would appreciate it if you could confirm it. 🙇

**Comment by m417z** [comment]
Can you post the crash report as you posted in your previous issue?
And, if possible, a video recording of full reproduction steps.

**Comment by yoyo-real** [comment]
Thanks for reply, I see.

Crash report here:
```
障害が発生しているアプリケーション名: explorer.exe、バージョン: 10.0.22000.778、タイム スタンプ: 0xaacb6d3d
障害が発生しているモジュール名: KERNELBASE.dll、バージョン: 10.0.22000.795、タイム スタンプ: 0x960371d1
例外コード: 0xc0000005
障害オフセット: 0x000000000004b469
障害が発生しているプロセス ID: 0x307c
障害が発生しているアプリケーションの開始時刻: 0x01d8a53ae181b1ad
障害が発生しているアプリケーション パス: C:\Windows\explorer.exe
障害が発生しているモジュール パス: C:\Windows\System32\KERNELBASE.dll
レポート ID: 5b95c0d1-7092-4333-a36a-b689d875eb3a
障害が発生しているパッケージの完全な名前: 
障害が発生しているパッケージに関連するアプリケーション ID: 
```


Video recording is possible too, but please allow a little time.

**Comment by m417z** [comment]
Should be fixed in Taskbar Thumbnail Reorder v1.0.2 (https://github.com/ramensoftware/windhawk-mods/pull/31). Let me know if the issue is not fixed.

**Comment by yoyo-real** [comment]
Solved!
Explorer hasn't hang with grouped thumbnail drag after update 🎉 
Very thanks for fix!
